### PR TITLE
Leaf/1.7 notifications table

### PR DESCRIPTION
**Pull request by sosropartono**
Notification Tables added, still cannot test the database, only provided SQL files

### CODE REVIEW

File: backend/settings.py [hunk 1]
```diff
@@ -85,8 +85,17 @@
...
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'user-data',
+        'HOST': '127.0.0.1',
+        'PORT': 3306,
+        'USER': 'root',
+        'PASSWORD': 'lmaozedong',
```
Comment (PattyCakesxD) [inline backend/settings.py:93]: Is this password meant to be here?

**Review by PattyCakesxD — APPROVED** [review]
Just 1 concern to be checked, lgtm otherwise

File: backend/settings.py (hunk 1 above)
Comment (sosropartono) [reply backend/settings.py:93]: Yeah, this doesn't seem like the best practice so going to remove this portion just so we have the SQL files.

**Review by misslame — CHANGES_REQUESTED** [review]
Update to use refactor code and make the model for the table using django

File: mesh/settings.py [hunk 2]
```diff
@@ -140,9 +140,3 @@
 
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.2/ref/settings/#default-auto-field
-
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
-
-STATICFILES_DIRS = [
```
Comment (misslame) [inline mesh/settings.py:146]: Why is this being deleted

**Review by misslame — CHANGES_REQUESTED** [review]

File: mesh/settings.py (hunk 2 above)
Comment (sosropartono) [reply mesh/settings.py:146]: Sorry about that, was debugging and accidentally deleted these lines.

File: mesh/notifications/models.py [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
 # Create your models here.
 
 class Notification(models.Model):
-    #TODO
-    pass
\ No newline at end of file
+    accountID = models.ForeignKey(Account, on_delete=models.CASCADE)
+    notificationID = models.AutoField(primary_key=True)
+    timeStamp = models.TimeField()
+    notificationContextJSON = models.JSONField()
```
Comment (peppacaiou) [inline mesh/notifications/models.py:10]: what is notficationContext supposed to be about?

Comment (sosropartono) [reply mesh/notifications/models.py:10]: I followed what was on the table but I think its just information about the notification like the message/type of notification?

**Pull request closed without merging**